fabric/bin/lib/common_ledger.sh

```diff
@@ -112,15 +112,32 @@ ledger_init() {
     try rm -rf ${FABRIC_STATE_DIR}/*
 
     # 2. start orderer
+    # try to create orderer-system-channel genesisblock
+    para
+    echo "[+] Creating a genesisblock for orderer-system-channel......."
+    para
+    try ${CONFIGTXGEN_CMD} -outputBlock genesisblock -profile SampleDevModeSolo -channelID orderer-system-channel
+    sleep 1
+
     ORDERER_GENERAL_GENESISPROFILE=SampleDevModeSolo ${ORDERER_CMD} 1>${ORDERER_LOG_OUT} 2>${ORDERER_LOG_ERR} &
     export ORDERER_PID=$!
     echo "${ORDERER_PID}" > ${ORDERER_PID_FILE}
     sleep 1
     kill -0 ${ORDERER_PID} || die "Orderer quit too quickly: (for log see ${ORDERER_LOG_OUT} & ${ORDERER_LOG_ERR})"
 
+
+
+
     # 3. start peer
-    LD_LIBRARY_PATH=${LD_LIBRARY_PATH:+"$LD_LIBRARY_PATH:"}${FPC_TOP_DIR}/tlcc/enclave/lib \
-		   ${PEER_CMD} node start 1>${PEER_LOG_OUT} 2>${PEER_LOG_ERR} &
+    para
+    echo "[-] NO LIBRARY IS BEING LOADED CURRENTLY....."
+    para
+    echo "[INFO] Just starting peer node directly.."
+    para
+    #LD_LIBRARY_PATH=${LD_LIBRARY_PATH:+"$LD_LIBRARY_PATH:"}${FPC_TOP_DIR}/tlcc/enclave/lib \
```

Comment:
Note the LD_LIBRARY_PATH environment doesn't cause any library to be loaded, only determines where libraries are found.  So you really shouldn't have to comment out this line ..

Reply:
In my testing when it is not commented out, it causes issues (breaks) .

Reply:
Can you be a bit more specific with the exact nature of the problem?  And could it maybe be that your FPC_TOP_DIR still points to the old 1.4 setup?

Reply:
Actually, testing now i realize that the issue has nothing to do with the LD_LIBRARY_PATH but that the peer dies in all cases! The reason why it seemed for you to work is because you removed the '&' and hence started the peer in the foreground. A properly running peer would actually have hang here but because it dies the script proceeds.  But obviously any subsequent peer action would fail!